demo/pages/checkable-rows/index.html

```diff
@@ -1,21 +1,23 @@
 <q-grid columns="$ctrl.columns"
 		  rows="$ctrl.rows"
 		  selection="$ctrl.selection"
-		  selection-mode="$ctrl.selectionMode"
+		  selection-mode="check"
```

Comment:
can we change it to selection-mode="row"?
i think for displaying checkbox we'll have other option?